src/utils.ts

```diff
@@ -1,13 +1,16 @@
 import type { Options } from './types'
 
 export const MIN_EVENTS_SYNC_INTERVAL = 60000
+export const MIN_POLLING_INTERVAL = 60000
 
 export const defaultOptions: Options = {
   debug: false,
   baseUrl: 'https://config.ff.harness.io/api/1.0',
   eventUrl: 'https://events.ff.harness.io/api/1.0',
   eventsSyncInterval: MIN_EVENTS_SYNC_INTERVAL,
+  pollingInterval: MIN_POLLING_INTERVAL,
   streamEnabled: true,
+  pollingEnabled: false,
```

Comment:
The more I think about this, the more I'm not sure. I think that _maybe_ if `streamEnabled` is `true`, then `pollingEnabled` should default to `true`. Unfortunately that means that logic is needed. I _think_ it might be a good idea to instead of just merging the defaults and the user-passed option in `index.ts`, instead have a function that can be called like this 👇🏻

```typescript
export const getConfiguration = (options: Options): Options => {
  const config = { ...defaultOptions, ...options }

  if (config.pollingEnabled === undefined) {
    config.pollingEnabled = config.streamEnabled
  }

  return config
}
```

So then in `index.ts`...
```typescript
// instead of
const configurations = { ...defaultOptions, ...options }

// you can do
const configurations = getConfiguration(options)
```

Reply:
Updated the readme